src/front-end/typescript/lib/pages/opportunity/code-with-us/lib/index.ts

```diff
@@ -44,6 +48,7 @@ export function cwuOpportunityStatusToTitleCase(
     case CWUOpportunityStatus.Canceled:
       return "Cancelled"; // Use British spelling for copy.
   }
+  return "";
```

Comment:
The switch cases are all covered so this statement unnecessary.

Reply:
ran into compiling errors and this was the only way through...though they may have been false negatives.

Reply:
taken care of in https://github.com/bcgov/digital_marketplace/pull/430/commits/d6eff27f9821cb4a669aa36c30115a7b995019a9